src/components/Modal/ConnectWalletModal/index.tsx

```diff
@@ -110,8 +110,9 @@ function ConnectWalletModal(props: ReactModal.Props) {
           !UNSUPPORT_WALLET_LIST[chainName].includes(wallet.info.name),
       )
       .map((wallet: BaseWallet) => {
-        const isInstalled = !!(
-          wallet.info.windowKey && wallet.info.windowKey in window
+        const isInstalled = !(
+          wm.getChainWalletState(wallet.info.name, chainName)?.walletState ===
+          WalletState.NotExist
         );
```

Comment:
```suggestion
        const isInstalled = 
          wm.getChainWalletState(wallet.info.name, chainName)?.walletState !==
          WalletState.NotExist;
```

nit; I think it's better for readibility

Reply:
https://github.com/dezswap/dezswap-web-app/pull/259/commits/49335b180ac0532d8a15e50a43eb11f44de1e456